docs/docsite/rst/scenario_guides/guide_docker.rst

```diff
@@ -43,12 +43,24 @@ Requirements
 ------------
 
 Using the docker modules requires having `docker-py <https://docker-py.readthedocs.io/en/stable/>`_
```

Comment:
Probably also want to change the link and the link text in this line.

Reply:
That's actually a good point. The link is correct (despite the `docker-py` in it's name), same as the Github repository for the library still being https://github.com/docker/docker-py/. Also, in the modules (both source and documentation), the library is usually referenced as `docker-py` (to distinguish it f.ex. from the `docker` executable). In fact, the `docker-py` authors admit (if you look at the checkmarks in this issue: docker/docker-py#1310) that they never finished the rename. It's really a big mess (and the main reason why all explanations tend to get long and ugly).

How about writing it as this (without mentioning `docker` or `docker-py`):
```
Using the docker modules requires having the `Docker SDK for Python <https://docker-py.readthedocs.io/en/stable/>`_
```

Reply:
That sounds great, thanks!

Reply:
Done. I also changed the `module` a bit below to SDK (since module could be understood as Ansible module).